docs/rvc/core_verification/checker_tasks.md

```diff
@@ -73,40 +73,40 @@ The tasks in this file keep a record of all the information stored in the regist
 # rf_write_history size     = 45
 #  >> rf_write_history[0] Mismatch!!
 # ** Error: ERROR: rf_write_history mismatch
-#    Time: 730 ps  Scope: big_core_rrv_tb.di_register_write File: ../../../verif/big_core_rrv/tb/mini_core_tasks.vh Line: 84
+#    Time: 730 ps  Scope: big_big_core_tb.di_register_write File: ../../../verif/big_big_core/tb/mini_core_tasks.vh Line: 84
```

Comment:
This looks strange

Reply:
I renamed it with error 
Note that big_core_rrv do not exists.
Its copy paste error